src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -113,27 +125,22 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
         ...node.subscription,
         ...(await createActivityContext())
     };
```

Comment:
you removed a bunch of files in this file. Please make sure to revert any changes that are not relevant to your work